tools/DevDiagnostics/DevHome.DevDiagnostics/Models/WERHelper.cs

```diff
@@ -51,7 +51,7 @@ public WERHelper()
         WERReports = new(_werReports);
 
         // Subscribe for Application events matching the processName.
-        EventLogQuery subscriptionQuery = new("Application", PathType.LogName, WERSubmissionQuery);
+        EventLogQuery subscriptionQuery = new EventLogQuery("Application", PathType.LogName, WERSubmissionQuery);
```

Comment:
Surprised the code analysis agent doesn't yell at you saying this could be written more simply....